lib/buffer.js

```diff
@@ -132,16 +139,29 @@ function createUnsafeBuffer(size) {
   }
 }
 
-function createPool() {
-  poolSize = Buffer.poolSize;
+// This is faster than using the bigint version.
```

Comment:
Is the difference measurable? If so, would just using `process.hrtime()[0]` and `now - last > 1` (amort 1.5) make sense? Probably not, asking just in case.